system/jlib/jsocket.cpp

```diff
@@ -1538,20 +1535,8 @@ void CSocket::connect_wait(unsigned timems)
     bool exit = false;
     int err;
     unsigned refuseddelay = 1;
-    while (!exit) {
-#ifdef CENTRAL_NODE_RANDOM_DELAY
-        ForEachItemIn(cn,CentralNodeArray) {
-            const SocketEndpoint &ep=CentralNodeArray.item(cn);
-            if (ep.ipequals(targetip)) {
-                unsigned sleeptime = getRandom() % 1000;
-                StringBuffer s;
-                ep.getHostText(s);
-                DBGLOG("Connection to central node %s - sleeping %d milliseconds", s.str(), sleeptime);
```

Comment:
I don't think I've ever seen this log message before ?
Probably there to avoid dropping SYNs when many connects at the same time ??
Increasing kernel somaxconn and application listen queues should help this.